src/Livewire/Mail/Mail.php

```diff
@@ -0,0 +1,143 @@
+<?php
+
+namespace FluxErp\Livewire\Mail;
+
+use FluxErp\Jobs\SyncMailAccountJob;
+use FluxErp\Livewire\DataTables\MailMessageList;
+use FluxErp\Livewire\Forms\MailMessageForm as MailMessageForm;
+use FluxErp\Models\MailAccount;
+use FluxErp\Models\MailFolder;
+use FluxErp\Models\MailMessage;
+use FluxErp\Models\Media;
+use Illuminate\Database\Eloquent\Builder;
+use Illuminate\Support\Arr;
+use Livewire\Attributes\Locked;
+use Symfony\Component\HttpFoundation\BinaryFileResponse;
+
+class Mail extends MailMessageList
+{
+    protected string $view = 'flux::livewire.mail.mail';
+
+    public array $folders = [];
+
+    public ?string $folderId = null;
+
+    public array $selectedFolderIds = [];
+
+    #[Locked]
+    public array $mailAccounts = [];
+
+    public MailMessageForm $mailMessage;
+
+    public function mount(): void
+    {
+        parent::mount();
+        $this->mailAccounts = auth()
+            ->user()
+            ->mailAccounts()
+            ->get(['mail_accounts.id', 'uuid', 'email'])
```

Comment:
isn't email also ambiguous?

Reply:
nope, tested. i think because the user is not fetched just the id (which is the same column name as in mail_accounts )